lib/github.js

```diff
@@ -71,6 +71,12 @@ Organization.prototype.getRepositories = function(callback) {
             cli.debug('next page ' + this.nextPageUrl);
             cli.debug('last page ' + this.getLastPage());
 
+            if ( body.message ) {
```

Comment:
Can you explain better? The error doesn't was catch [here](https://github.com/tegon/clone-org-repos/pull/3/files#diff-8f83cf4e554cfef336da48f2169e3704R64)?